packages/fxa-shared/subscriptions/types.ts

```diff
@@ -177,7 +178,7 @@ export enum SubscriptionEligibilityResult {
 }
 
 export type SubscriptionUpdateEligibility =
-  typeof SubscriptionUpdateEligibility[keyof typeof SubscriptionUpdateEligibility];
+  (typeof SubscriptionUpdateEligibility)[keyof typeof SubscriptionUpdateEligibility];
```

Comment:
Did you make this change manually or did it happen on auto-save?

Reply:
Oh I didn't even notice this changed. Must have been on auto-save.

Reply:
A similar thing happened with Ivo's PR last week. Appears to bug with the auto-save and should be reverted as the two should not be equivalent.

Reply:
Ended up looking into this a bit and @julianpoy mentioned the following.

> The corrected type is desired. It’s being updated to match the JS syntax equivalent for keyof, since typescript treats both (typeof A)[keyof typeof A] the same as typeof A[keyof typeof A]

With that in mind, I'll leave the edit as is.